acceptance/suites/pre_suite/foss/00_setup_environment.rb

```diff
@@ -1,2 +1,13 @@
-step "Initialize Test Config"
+step "Initialize Test Config" do
   PuppetServerExtensions.initialize_config options
+
+  if PuppetServerExtensions.config[:puppet_build_version] == 'LATEST'
```

Comment:
I've replaced the default values with `nil` when getting the version option values, could you take a look at the latest changes?